src/main/headless.ts

```diff
@@ -10,6 +10,7 @@ import { EventEmitter } from "ws";
 import { BlockMetadata } from "src/interfaces/block-header";
 import { KeyStore } from "./headless/key-store";
 import { Validation } from "./headless/validation";
+import { Apv } from "./headless/apv";
```

Comment:
if created 'headless' directory, we should move `src/main/headless.ts` into that directory.

Reply:
Applied. https://github.com/planetarium/9c-launcher/pull/640/commits/f6c9fb81d7fd4a91cbd49de0fc134ec870fabb3c